docs/reference/mapping/params/copy-to.asciidoc

```diff
@@ -67,6 +67,13 @@ Some important points:
 * You cannot copy recursively via intermediary fields such as a `copy_to` on 
 `field_1` to `field_2` and `copy_to` on `field_2` to `field_3` expecting 
 indexing into `field_1` will eventuate in `field_3`, instead use copy_to 
-directly to multiple fields from the originating field. 
+directly to multiple fields from the originating field.
+* If the target field does not exist in the mapping, the usual
+<<dynamic-mapping,dynamic mapping>> behavior applies. By default, with
+<<dynamic,`dynamic`>> set to `true`, a non-existent target field will be
+dynamically added to the document and the mapping. If `dynamic` is set to
```

Comment:
to the mapping. I think that if you say "to the document" people may understand that we modify `_source`. But you are right, it is added to the lucene document.

Reply:
I've changed it into "a non-existent target field will be dynamically added to the index mappings"